### PR TITLE
MAP-1036 add GET archived endpoint

### PR DESCRIPTION
**Pull request by GurnankCheema**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt [hunk 1]
```diff
@@ -30,6 +30,7 @@ class Cell(
   comments: String? = null,
   orderWithinParentLocation: Int? = null,
   active: Boolean = true,
+  archived: Boolean = false,
```
Comment (Mjwillis) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt:33]: not needed

Comment (Mjwillis) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt:34]: Still not needed

File: src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/ResidentialLocationRepository.kt [hunk 2]
```diff
@@ -3,10 +3,11 @@ package uk.gov.justice.digital.hmpps.locationsinsideprison.jpa.repository
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.locationsinsideprison.jpa.ResidentialLocation
-import java.util.UUID
```
[the remark this answers is not in the document]
Comment (GurnankCheema) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/repository/ResidentialLocationRepository.kt:6]: intelli j did it automatically!

File: src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt (hunk 1 above)
Comment (GurnankCheema) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Cell.kt]: inadvertently missed

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**